### ISSUE TITLE
Label widget doesn't work second time

### ISSUE THREAD

**Issue by jasongrout** [ipython/ipywidgets#550]
In the Widget List example notebook, there is an example of the Label widget:

```python
widgets.Label(
    value="$$\\frac{n!}{k!(n-k)!} = \\binom{n}{k}$$",
)
```
The first time I run the cell, it looks fine. If I run it a second time, though, I just get "Math processing error". Right-clicking on the error and selecting "Error message" says "Error: Cannot read property 'firstChild' of null"

CC @afshin.

**Comment by jasongrout** [comment]
I tracked this down to an error in the MathJax PreviewHTML code: it looks like this line: https://github.com/mathjax/MathJax/blob/d4ab1b35c96dd964eaa9e1ed2c86e39fffbdacf6/unpacked/jax/output/PreviewHTML/jax.js#L511 is returning `null`, which is returned to https://github.com/mathjax/MathJax/blob/d4ab1b35c96dd964eaa9e1ed2c86e39fffbdacf6/unpacked/jax/output/PreviewHTML/jax.js#L666. Later in that function, `span.firstChild` is used at https://github.com/mathjax/MathJax/blob/d4ab1b35c96dd964eaa9e1ed2c86e39fffbdacf6/unpacked/jax/output/PreviewHTML/jax.js#L680, which causes the complaint that null doesn't have a firstChild property.

@afshin - don't worry about this for now. I'll report it upstream to MathJax.

**Comment by jasongrout** [comment]
This may be related to https://github.com/mathjax/MathJax/issues/1185. Basically, the preview html renderer *needs* the content to be in the page. We should check that anytime we call typeset, the element is actually in the DOM. Perhaps we are asking MathJax to typeset an element that has been created, but hasn't quite been inserted into the DOM yet. Perhaps we should use a `this.displayed().then(...)` sort of construct for the typesetting call.

I reproduced this on Sylvain's bqplot binder (with the Latex widget), for example, using MathJax 2.6.3, so it's not just affecting master.

**Comment by SylvainCorlay** [comment]
Thanks for looking into this @jasongrout !
